### ISSUE TITLE
Will it work for characters?

### ISSUE THREAD

**Issue by MarStarck** [potterhsu/SVHNClassifier#18]
I wanna recognize images including short characters as well as digits, like "A25", "B03" and so on. I wonder could this structure work on this situation?

**Comment by fkoorc** [comment]
Looks like the training data only have digital, i test and characters is not working
If you want character you have to add character training data

**Comment by MarStarck** [comment]
I have added character training data but the acc drops to about 70%

**Comment by fkoorc** [comment]
Actually my model only have digital and still the test accuracy is about 75%
but if the picture is No distortion and very clear (maybe use some pre-process OCR) the accuracy can be about 93% I guess the model was overfitting.
Or maybe your training data format had some problem may I ask your character data source?

**Comment by MarStarck** [comment]
the data source is synthetic text data by [TextRecognitionDataGenerator](https://github.com/Belval/TextRecognitionDataGenerator)
I believe they are quite clear without distortion. So I think the 10-layer conv model cannot describe these characters, it's under fitting.

**Comment by fkoorc** [comment]
I guess you are right.
I saw the data contains en/cn/fr/de?  Looks quite complicate. If your data is large enough. I think 70% is quite good result 
If you still wanna promote accuracy, I suggest you can see these two paper

Synthetic Data and Artificial Neural Networks for Natural Scene Text Recognition（2014）
Reading Text in the Wild with Convolutional Neural Networks（2014）


Where are you from? Maybe we can change email and discuss

**Comment by MarStarck** [comment]
@fkoorc I'm from China, my email: difei_coder@163.com
